apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/evaluations/[evaluationId]/_components/MetricsSummary/BigNumberPanels/ModalValuePanel/index.tsx

```diff
@@ -30,7 +31,11 @@ export default function ModalValuePanel({
       fallbackData: modal,
     },
   )
-  const onStatusChange = useCallback(() => refetch(), [refetch])
+  const onStatusChange = useDebouncedCallback(
+    useCallback(() => refetch(), [refetch]),
+    2000,
+    { trailing: true },
+  )
```

Comment:
absolutely bonkers this was recalculating on each new eval result coming in real time resulting in a self-ddos

Reply:
If we have realtime maybe these heavy computational stats should be left stale forever by configuring the SWR store no?

Reply:
yeah probably

Reply:
Do this please. I think it makes a lot of sense